### ISSUE TITLE
Validation loss and acc don't show

### ISSUE THREAD

**Issue by AliaMYH** [NVIDIA/DIGITS#1488]
![screen shot 2017-03-02 at 3 59 23 pm](https://cloud.githubusercontent.com/assets/10338687/23510227/565f9a3e-ff61-11e6-8b18-e6afeeedf13e.png)
 
I'm trying to fine-tune squeeze net and i've edited the deploy.prototxt but only the training values are shown, not the validation. My validation set is a fraction of my testing set.

The following are the final layers of my network.
....
layer {
  name: "relu_conv10"
  type: "ReLU"
  bottom: "conv10_new"
  top: "conv10_new"
}
layer {
  name: "pool10"
  type: "Pooling"
  bottom: "conv10_new"
  top: "pool10"
  pooling_param {
    pool: AVE
    global_pooling: true
  }
}
layer {
  name: "loss"
  type: "SoftmaxWithLoss"
  bottom: "pool10"
  bottom: "label"
  top: "loss"
  include {
    phase: TRAIN
  }
}
layer {
  name: “softmax”
  type: "Softmax"
  bottom: "pool10"
  top: “softmax”
  exclude {
    phase: TRAIN
  }
}
layer {
  name: "accuracy"
  type: "Accuracy"
  bottom: "pool10"
  bottom: "label"
  top: "accuracy"
  include {
    phase: TRAIN
  }
}
layer {
  name: "accuracy_top5"
  type: "Accuracy"
  bottom: "pool10"
  bottom: "label"
  top: "accuracy_top5"
  include {
    phase: TRAIN
  }
  accuracy_param {
    top_k: 5
  }
}

**Comment by AliaMYH** [comment]
Update: I commented out all the includes and the same thing is still happening

**Comment by AliaMYH** [comment]
I'm a bit confused about the difference between a 'phase' and a 'stage'; phase: TRAIN and stage: deploy, for example. 
@lukeyeager if you could clarify, that would be great.

Thank you

**Comment by lukeyeager** [comment]
The documentation for this can be accessed when creating a custom network. See this screenshot: https://github.com/NVIDIA/DIGITS/pull/628#issuecomment-195137527
